tenable/sc/organization.py

```diff
@@ -76,31 +101,117 @@ def _constructor(self, **kw):
+                Valid choices are ``auto_only``, ``locked``, ``selectable``,
+                ``selectable+auto``, ``selectable+auto_restricted``.
+            zones (list, optional):
+                When ``zone_selection`` is not ``auto_only``, this field
+                must be filled with list of id from available scan zone(s).
+            restricted_ips (str, optional):
+                A valid IP address or range of IP addresses
+            lce_correlation (list, optional):
+                What Log Correlation Engines (if any) which correlate against
+                this organization.  A list of configured LCE numeric IDs is
+                supplied.  This option is used on IPv4, IPv6, and agent formats
+                and is defaulted to nothing if left unspecified.
+            repo (list, optional):
+                A list of Repositories to which the Organization will have
+                access.
+            pub_sites (list, optional):
+                Organizations may configure publishing sites as targets
+                to send report results to a properly configured web server
+                or a Defense Information Systems Agency (DISA)
+                Continuous Monitoring and Risk Scoring (CMRS) site.
+            ldaps (list, optional):
+                Accessible LDAP Servers.
+            nessus_managers (list, optional):
+                Nessus Manager scanner for Nessus Agent scan imports.
+            vuln_score_low (int):
+                The vulnerability weighting to apply to “Low” criticality
+                vulnerabilities for scoring purposes. (Default: 1)
+            vuln_score_medium (int):
+                The vulnerability weighting to apply to “Medium” criticality
+                vulnerabilities for scoring purposes. (Default: 3)
+            vuln_score_high (int):
+                The vulnerability weighting to apply to “High” criticality
+                vulnerabilities for scoring purposes. (Default: 10)
+            vuln_score_critical (int):
+                The vulnerability weighting to apply to “Critical” criticality
+                vulnerabilities for scoring purposes.(Default: 40)
+            ip_info_links (list, optional):
+                A list of custom analysis links provided to users within the
+                host vulnerability details when analyzing data outside of
+                SecurityCenter is desired.
+
+        Returns:
+            dict: The organization resource record for the newly created
+                  Organization.
+
+        Examples:
+            Creating a new organization with automatic scan zone selection:
+
+            >>> org = sc.organization.create(
+            ...     name="Sample Organization",
+            ...     zone_selection="auto_only")
+
+            Creating a new organization with custom analysis links:
+
+            >>> org = sc.organization.create(
+            ...     name="Sample Organization",
+            ...     ip_info_links=[
```

Comment:
What are your thoughts on the info links being tuples?  Instead of passing `[dict(name='SANS', link='LINK')]` as you are, it could be passed as [('SANS', 'LINK')].  A bit more compact, no real loss of information, and the param that would expand it would be simply "links".

Reply:
it is a lot more nicer I thinks. i'll change that